e2e-tests/playwright/tests/functional/channels/edit_file_attachments/edit_file_attachment.spec.ts

```diff
@@ -0,0 +1,351 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {test} from '@e2e-support/test_fixture';
+import {expect} from '@playwright/test';
+import {duration, wait} from '@e2e-support/util';
+import {moveMouseToCenter} from '@e2e-support/test_action';
+
+test('MM-T5654_1 should be able to add attachments while editing a post', async ({pw}) => {
```

Comment:
Just curious: I don't see tests for steps MM-T5654_2 and MM-T5654_6. Is that intended and for follow up?